src/index.ts

```diff
@@ -39,7 +39,7 @@ export interface CheckoutManifest {
 export type CheckoutType = 'single' | 'frequent' | 'subscription'
 
 /** The possible payment methods. */
-export type CheckoutMethod = 'cc' | 'mbw' | 'mb' | 'dd' | 'vi' | 'uf' | 'sc'
+export type CheckoutMethod = 'cc' | 'mbw' | 'mb' | 'dd' | 'vi'
```

Comment:
Since we're doing this, let's take the opportunity to add `ap` and `gp` for the sake of completion.